pkg/registry/registry.go

```diff
@@ -341,18 +340,14 @@ func (r *Registry) handleBlob(rw mux.ResponseWriter, req *http.Request, ref refe
 	if req.Method == http.MethodHead {
 		return
 	}
-	var w io.Writer = rw
+	var w mux.ResponseWriter = rw
```

Comment:
This can be removed and we can pass rw right into http.ServeContent. I forgot to remove this while doing some refactoring.